app/templates/index.html

```diff
@@ -25,71 +33,66 @@ <h1 class="panel__title venus">
+    <div class="panel panel--simple panel--spacious {% if messages %} panel--error {% else %} panel--info {% endif %} u-mb-l">
       <div class="panel__body">
-
         <div class="group" id="household-and-accommodation">
-
           <div class="block" id="census-access-code">
             <div class="section" id="census-access-code-section">
               <div class="question" id="census-access-code-question">
-                <div class="panel panel--simple panel--neutral" id="question-guidance-census-access-code-question">
-                  <div class="question__answers">
-                    <div class="question__answer">
-                      <div class="answer answer--integer " id="census-access-code-answer">
-                        <div class="answer__fields js-fields">
-                          <fieldset class="fieldgroup fieldgroup--date">
-                            <legend class="fieldgroup__title venus">Enter your unique access code below</legend>
-                            <div class="fieldgroup__fields">
-                              <div class="field field--input field--code">
-                                <label class="label mercury u-vh" for="iac1">Enter the first four characters of the access code</label>
-                                <input pattern="^[a-zA-Z0-9]{4}$" maxlength="4" data-qa="input-text" id="iac1" name="iac1" class="input input--text" value=""
-                                  type="text" autocomplete="off" required>
-                              </div>
-                              <div class="field field--select field--code">
-                                <label class="label mercury u-vh" for="iac2">Enter the next four characters of the access code</label>
-                                <input pattern="^[a-zA-Z0-9]{4}$" maxlength="4" data-qa="input-text" id="iac2" name="iac2" class="input input--text" value=""
-                                  type="text" autocomplete="off" required>
-                              </div>
-                              <div class="field field--input field--code">
-                                <label class="label mercury u-vh" for="iac3">Enter the last four characters of the access code</label>
-                                <input pattern="^[a-zA-Z0-9]{4}$" maxlength="4" data-qa="input-text" id="iac3" name="iac3" class="input input--text" value=""
-                                  type="text" autocomplete="off" required>
-                              </div>
+                <div class="question__answers">
+                  <div class="question__answer">
+                    <div class="answer answer--integer " id="census-access-code-answer">
+                      <div class="answer__fields js-fields">
+                        <fieldset class="fieldgroup fieldgroup--date">
+                          <legend class="fieldgroup__title venus">To start or continue the study, enter your 12 digit household access code and select "Start now".</legend>
```

Comment:
character -> digit? UAC is still 12 characters as far as I know. (And the input patterns below are still alpha numeric).